_search-query/text-analyzers.md

```diff
@@ -1,12 +1,12 @@
 ---
 layout: default
-title: Text analyzers
-parent: Query DSL
-nav_order: 41
+title: Optimize text for searches
```

Comment:
Make sure that we make a directory for this page and all of this page's children.

Reply:
ok i can do that again. that's how i had it prior to the single menu feedback. 

> We need to clean up the directory structure of this PR before we merge it in. The changes include:
> 
> * Move any pages under the `_opensearch` directory that have been changed to the `_search-and-query` directory.
This was done after the first round of feedback you provided. 
Will do for the new ones ( Aggregate data for analysis category).
> * Create sub-directories under `_search-and-query` for each Parent section, including: "Asynchronous search plugin", "Search using Query DSL", "Search using kNN", and "Prepare data for searches".
Got it. Will build those out and move to Search
> * Move any edited pages from `_search-plugins` into the `_search-and-query` directory.
> * Remove `search-plugins` from collections.
Yes.
Sounds good. I can build the alternate structures you suggested.